modules/auth/cms_authentication_module.e

```diff
@@ -91,12 +91,12 @@ feature -- Router
 
 feature -- Hooks configuration
 
-	register_hooks (a_response: CMS_RESPONSE)
+	setup_hooks (a_hooks: CMS_HOOK_CORE_MANAGER)
 			-- Module hooks configuration.
```

Comment:
Maybe --<Precursor> and move Module hooks configuration description to CMS_MODULE.setup_hooks.